content/docs/06-integrations/00-byoos.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "BYOOS"
+metaTitle: "Bring your own OS"
+metaDescription: "Customize your Edge host deployment with your own Operating System."
+hiddenFromNav: true
+type: "integration"
+category: ["byoos"]
+logoUrl: ""
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Bring Your Own Operatin System (BYOOS) 
+
+Bring Your Own Operating System (BYOOS) is a deployment option that allows you to customize the operating system of their environment. 
+This BYOOS feature optimizes infrastructure and streamlines workflow by enabling users to specify a host runtime operating system (OS), providing greater flexibility and control over their infrastructure. 
+
+BYOOS empowers you to deploy your preferred OS to your environments and tailor them to your specific needs using various customization tools. 
+
+Spectro Cloud offers BYOOS support for both Edge and non-Edge environments, allowing you to customize your environment's operating system (OS). You can reference the BYOOS pack in Terraform using a data resource, which can help optimize their workflow. 
+
+Check out the guides to how to use BYOOS in Edge and Non-Edge.
+
+# Versions Supported
+
+**1.0.x**
+
+# Prerequisites 
+
+To use the Non-Edge BYOOS pack, you must have the following:
+
+- Access to a Non-Edge repository that contains the generic-byoi pack
```

Comment:
```suggestion
- Access to a non-Edge repository that contains the generic BYOOS pack.
```

Reply:
@rahulhazra97 Thanks for adding the integration doc! According to our Packs Layout, this doc needs the following sections:

**Parameters**
**Usage** - this could be the section to explain some of the specific configuring users can do.
**Terraform**
**Resources**

For an example, you can look at https://docs.spectrocloud.com/integrations/kubernetes. 
Packs Layout: https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1800994817/Packs+Layout